scripts/styleguide.setup.js

```diff
@@ -8,5 +8,40 @@ global.FOLDER_ID = global.FOLDER_ID || __FOLDERID__ || '69083462919'; // eslint-
 global.TOKEN = global.TOKEN || __TOKEN__ || 'P1n3ID8nYMxHRWvenDatQ9k6JKzWzYrz'; // eslint-disable-line
 global.PROPS = global.PROPS || {}; // eslint-disable-line
 
-// Components
+// Classes
 global.IntlProvider = IntlProvider;
+
+// Components
+global.Avatar = require('../src/components/avatar').default;
+global.Button = require('../src/components/button').default;
+global.DatalistItem = require('../src/components/datalist-item').default;
+global.Flyout = require('../src/components/flyout').Flyout;
+global.Label = require('../src/components/label').default;
+global.Link = require('../src/components/link').default;
```

Comment:
you should check double check if these have defaults. i dont think `Link` has a default

Reply:
it does, otherwise it would have broke in the styleguidist, we went down the list one by one and all the components you see there were cherry picked

Reply:
it's exporting the defaults as named exports tho. there's no default being exported out of that index right?

Reply:
done